src/gui/plugins/resource_spawner/ResourceSpawner.cc

```diff
@@ -255,29 +426,67 @@ void ResourceSpawner::LoadConfig(const tinyxml2::XMLElement *)
   if (!executed || !result || res.data_size() < 1)
   {
     ignwarn << "No paths found in IGN_GAZEBO_RESOURCE_PATH.\n";
-    return;
   }
 
   for (int i = 0; i < res.data_size(); i++)
   {
     const std::string path = res.data(i);
     this->AddPath(path);
   }
+
+  auto servers = this->dataPtr->fuelClient->Config().Servers();
+  QGuiApplication::setOverrideCursor(Qt::WaitCursor);
+  ignmsg << "Please wait... Loading fuel models from online.\n";
```

Comment:
How about something like

```suggestion
  ignmsg << "Please wait... Loading models from Fuel.\n";
```

Reply:
Resolved fdff6c8149c682c4141e5110e3b31d8618730867